doc/free-func.rst

```diff
@@ -344,8 +344,88 @@ and only then the final solid can be constructed.
     
     # local sewing - only two faces are taken into account
     sh = shell(b_top_hole, feat.faces('<Z'), ctx=(b, feat))
-    
     # construct the final solid
     result = solid(sh)
 
 
+Mapping onto parametric space
+-----------------------------
+
+To complement functionalities described It is possible to trim edges and faces explicitly using simple rectangular
+trims, polygons, splines or arbitrary wires.
```

Comment:
Grammar.  Either:
_It is possible to trim edges and faces explicitly using simple rectangular trims, polygons, splines or arbitrary wires._

or add comma and change **It** to lower case:
_To complement functionalities described, it is possible to trim edges and faces explicitly using simple rectangular trims, polygons, splines or arbitrary wires._